packages/react-dev-utils/README.md

```diff
@@ -332,7 +332,8 @@ Creates a Webpack compiler instance for WebpackDevServer with built-in helpful m
 The `args` object accepts a number of properties:
 
 - **appName** `string`: The name that will be printed to the terminal.
-- **config** `Object`:  The webpack configuration options to be provided to the webpack constructor.
+- **appPath** `string`: The path to the application. Used to make error / warning paths pithier.
```

Comment:
Typo

Reply:
what's the typo?  Looks good to me?  I can confirm that "pithier" is a word 😄  https://en.wiktionary.org/wiki/pithier